src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -706,7 +706,7 @@ namespace
     {
         bool success = true;
 
-        if (extension != ".exr")
+        if (extension != ".exr" && extension != ".EXR")
```

Comment:
That's not robust enough (wouldn't work with ".Exr" for instance). I suggest converting the extension to lower case before doing the check.

Reply:
Ok, then we should also do that in write_image

Reply:
It's already properly done in `write_image()`:

    string extension = lower_case(bf_file_path.extension().string());

Reply:
That said, the problem appears to also exist in `write_main_image()`, maybe that's what you meant.

Reply:
No it wasn't, I haven't seen the lower_case code. But thanks for pointing out this, I will fix that